source/dbo/Views/Power BI Reporting/vwUnmatchedDuplicates.sql

```diff
@@ -0,0 +1,82 @@
+﻿CREATE VIEW [dbo].[vwUnmatchedDuplicates]
+AS
+
+-- NTBS Notifications, with all notifications migrated in from ETS
+WITH NtbsNotifications AS (
+	SELECT n.NotificationId AS NtbsId,
+		CAST(n.ETSID AS INT) AS EtsId,
+		n.GroupId AS NtbsGroupId,
+		etsGroupings.GroupId AS EtsGroupId,
+		n.NotificationDate,
+		n.CreationDate,
+		p.Dob,
+		p.FamilyName,
+		REPLACE(p.NhsNumber, ' ', '') AS NhsNumber,
+		u.Username AS CaseManager
+	FROM [$(NTBS)].dbo.[Notification] n
+		INNER JOIN [$(NTBS)].dbo.Patients p ON n.NotificationId = p.NotificationId
+		INNER JOIN [$(NTBS)].dbo.HospitalDetails h ON n.NotificationId = h.NotificationId
+		LEFT JOIN [$(NTBS)].dbo.[User] u ON h.CaseManagerId = u.Id
+		LEFT JOIN [$(migration)].dbo.EtsGroupings etsGroupings ON n.ETSID = etsGroupings.EtsId
+	WHERE n.NotificationStatus IN ('Notified', 'Closed')
+),
+-- ETS notifications that have not been migrated into NTBS
+EtsNotifications AS (
+	SELECT NULL AS NtbsId,
+		n.LegacyId AS EtsId,
+		NULL AS NtbsGroupId,
+		etsGroupings.GroupId AS EtsGroupId,
+		n.NotificationDate,
+		n.AuditCreate AS CreationDate,
+		p.DateOfBirth,
+		p.Surname AS FamilyName,
+		REPLACE(p.NhsNumber, ' ', '') AS NhsNumber,
+		su.Username AS CaseManager
+	FROM [$(ETS)].dbo.[Notification] n
+		INNER JOIN [$(ETS)].dbo.Patient p ON n.PatientId = p.Id
+		INNER JOIN [$(ETS)].dbo.SystemUser su ON n.OwnerUserId = su.Id
+		LEFT JOIN [$(NTBS)].dbo.[Notification] ntbsNotification ON n.LegacyId = ntbsNotification.ETSID
+		LEFT JOIN [$(migration)].dbo.EtsGroupings etsGroupings ON n.LegacyId = etsGroupings.EtsId
+	WHERE n.AuditDelete IS NULL AND n.Submitted = 1 AND n.DenotificationId IS NULL
+		AND ntbsNotification.NotificationId IS NULL -- ie no corresponding NTBS migrated notification
+),
+AllNotifications AS (
+	SELECT *
```

Comment:
Worth including a SourceSystem here for debugging purposes?  It would also prevent two identical records from the two different systems being collapsed into one record by the UNION.  That should obvs never happen anyway.